k8s/base/kustomization.yaml

```diff
@@ -1,10 +1,16 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: animal-rescue
+secretGenerator:
+- name: gateway-sso-credentials
+  envs:
+  - secrets/test-sso-credentials.txt
+  type: Opaque
```

Comment:
I thought the SSO credentials secret needs to be in the SCG installation namespace?